publishers/HumanStatePublisher/HumanStatePublisher.cpp

```diff
@@ -145,6 +153,26 @@ bool HumanStatePublisher::open(yarp::os::Searchable& config)
         return false;
     }
 
+
+    // =========================
+    // OPEN THE TRANSFORM CLIENT
+    // =========================
+
+    yarp::os::Property options;
+    options.put("device", "transformClient");
+    options.put("local", "/" + DeviceName + "/transformClient");
+    options.put("remote", "/transformServer");
```

Comment:
The transform client should be opened using the `portprefix` if `hasPortPrefix` is trure (like in https://github.com/robotology/human-dynamics-estimation/blob/devel/publishers/HumanStatePublisher/HumanStatePublisher.cpp#L130)

Reply:
@lrapetti the changes in this PR are based on the `master` branch. So, for the time being we can merge it to master and while doing an update on the master from `devel` branch the updates will be done correctly.

Reply:
you are right, we should remember to add it when moving to `devel`